### ISSUE TITLE
App only enters Jellyfin GUI

### ISSUE THREAD

**Issue by pinguin-82bit** [leinelissen/jellyfin-audio-player#215]
I'm using an IPhone. When I'm entering the app and type in my server adress, then the app just enters the Jellyfin GUI.
I tried the same on my old Android device and there it also entered the Jellyfin GUI but after a few seconds it entered the Fintunes GUI and everything worked fine.

It works now. I tried it a few more times and mow it works.

**Comment by aethrvmn** [comment]
I have the same problem right now

**Comment by leinelissen** [comment]
Hey that's weird and shouldn't be happening. Could you check which Fintunes version you're running and potentially try the [latest beta](https://testflight.apple.com/join/TpHWMmfM)?
If that doesn't help, debugging your specific server setup might help me out. If you feel comfortable sharing some credentials with me over Discord or e-mail, I can take a look.

**Comment by aethrvmn** [comment]
Everything is run locally only so its not possible to share any credentials, The beta has the same issue. My server is pretty basic in terms of plugins, and settings in general since I only set it up on Friday. I will try to remove/play around and come back hopefully with good news.

**Comment by pinguin-82bit** [comment]
When i tried accessing over the same Network in which the server runs, it didn't work but when i tried to connect via a WireGuard tunnel, it somehow worked.

**Comment by leinelissen** [comment]
Hmm, that's interesting. None of it should make a difference. All the app tries is to fetch any credentials from the localstorage web view, parse them and verify they work. All the code is in this file: https://github.com/leinelissen/jellyfin-audio-player/blob/b03d449a1bd5d16e16f15dbb215a75cfd2e0cc7c/src/screens/modals/SetJellyfinServer/components/CredentialGenerator.tsx#L60-L153).

Both of you could attempt to run `JSON.parse(window.localStorage.getItem('jellyfin_credentials'))` in your browser window consoles with the given network settings and see if the outputs differ between network setups.

**Comment by aethrvmn** [comment]
Today it decided to work before I tried absolutely anything on the server???

**Comment by LordVicky** [comment]
I just experienced this issue, I setup jellyfin on a new server with default podman ports, made sure it was added to firewall and everything works as expected. But when I add the server to the Fintunes app, it didn't recognize the user. I tried wiping web data in my iphone, checked jellyfin logs but everything seemed fine.

I even tried other apps like FinAmp, which works perfectly, I also tried adding back my old server but it didn't get authenticated exactly like the OP says.

**I finally tried restarting the phone** to see if the cache is cleared and voila! it works. I can't reproduce the issue either so it is pretty hard to debug, Do let me know  if you require any logs or my podman-compose file for debugging. I really appreciate the work put into this, if I may suggest, a manual user authentication method like Fin-Amp has would be great for solving this issue.
